torch/csrc/jit/tensorexpr/schedule.cpp

```diff
@@ -392,6 +392,203 @@ ScheduleObject* ScheduleNode::CloneScheduleObject(ScheduleObject* object) {
   return new_object;
 }
 
+class Vectorizer : public IRMutator {
+  public:
+  Vectorizer(const Var& v, int lanes) : var_(v), lanes_(lanes) {}
+
+  Expr mutate(const Add* v) override {
+    std::vector<Expr> inputs = { v->lhs(), v->rhs() };
+    return try_vectorize(Expr(v), inputs,
+      [&](){ return inputs[0] + inputs[1]; });
```

Comment:
Minor nit: I personally would like to see what is actually captured, if the capture list is not too long.